pkg/devfile/parser/data/2.0.0/types.go

```diff
@@ -23,388 +15,17 @@ type Devfile200 struct {
 	Components []common.DevfileComponent `json:"components,omitempty"`
 
 	// Bindings of commands to events. Each command is referred-to by its name.
-	Events common.DevfileEvents `json:"events,omitempty"`
+	Events apiComp.WorkspaceEvents `json:"events,omitempty"`
 
 	// Optional metadata
 	Metadata common.DevfileMetadata `json:"metadata,omitempty"`
```

Comment:
why some fields are still referring to `common`?

Reply:
Dockerfile component is to be part of devfile 2.1.0 schemaVersion, and since devfile/api doesn't have dockerfile component added yet, it is defined within the library. Thus the components are still wrapped as DevfileComponent defined in common to support both 2.0.0 and 2.1.0 from the library, untill dockerfile component is added on the devfile/api and supports other 2.1.0 components.